h/templates/accounts/account.html.jinja2

```diff
@@ -41,6 +42,23 @@
             {{ svg_icon('external', 'external-icon') }}
           </a>
         {% endif %}
+        {% endif %}
+
+        {% if log_in_with_google %}
+        {% if google_id %}
+          <a class="sso-connect" target="_blank" rel="nofollow noopener">
```

Comment:
No `href` here. AFAIK Google doesn't have a public profile page to link to like ORCID does.